ocs_ci/ocs/ui/views.py

```diff
@@ -1226,6 +1231,15 @@
         By.XPATH,
     ),
     "button_with_txt": ("//button[text()=('{}')]", By.XPATH),
+    "volume_binding_mode": (
+        "button[id='storage-class-volume-binding-mode']",
+        By.CSS_SELECTOR,
+    ),
+    "immediate_binding_mode": ("button[id='Immediate-link']", By.CSS_SELECTOR),
+    "storage_class_breadcrumb": (
+        "//a[@class='pf-c-breadcrumb__link'][text()='StorageClasses']",
```

Comment:
This locator won't work in 4.16, after patternfly 5 changes. Could you please re-check the locators for a 4.16 cluster?

Reply:
Sure will check it out

Reply:
the tests are working fine

Reply:
Could it be that you're not actually using this locator in the tests?

Reply:
yea I see its not used, will remove storage_class_breadcrumb locater